kuksa_apps/node-red/docker-compose.yml

```diff
@@ -4,6 +4,7 @@ services:
   mynodered:
     image: nodered/node-red
     restart: unless-stopped
+    user: root
```

Comment:
@anntexx this seems also fixt your problem, so that it is not necessary to add the user to docker group. It may be a better solution

Reply:
+1 then we will not document the work around of adding docker to the user group